Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM golang:1.4
+FROM golang:1.7
 
 ADD . /app/src/github.com/Shopify/toxiproxy
-RUN cd /app/src/github.com/Shopify/toxiproxy && GOPATH=/app/src/github.com/Shopify/toxiproxy/Godeps/_workspace:/app go build -ldflags="-X github.com/Shopify/toxiproxy.Version $(cat VERSION)" -o /app/toxiproxy ./cmd
+
+ENV GOPATH /app:$GOPATH
```

Comment:
Will this actually use the Godeps workspace? I'm not sure what the default $GOPATH is here.